# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-am-build ncurses-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.71; am-1.16)
aclocal -I m4; autoheader; autoconf; automake -a -c
./configure --prefix=

# build
make

# install
install -d $DESTDIR/bin
install src/fuser $DESTDIR/bin
install src/killall $DESTDIR/bin
install src/peekfd $DESTDIR/bin
install src/prtstat $DESTDIR/bin
install src/pslog $DESTDIR/bin
install src/pstree $DESTDIR/bin

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/psmisc-build.pc << EOF
Name: psmisc
Version: 23.6
Description: Miscellaneous utilities regarding running processes
Requires: $REQUIRES

exec=\\
/bin/fuser \\
/bin/killall \\
/bin/peekfd \\
/bin/prtstat \\
/bin/pslog \\
/bin/pstree
EOF

exit

# patch
--- psmisc-23.6.orig/configure.ac
+++ psmisc-23.6/configure.ac
@@ -3,6 +3,7 @@
 AC_CONFIG_MACRO_DIR([m4])
 AC_INIT([psmisc],[m4_esyscmd(misc/git-version-gen .tarball-version)],[],[],[],[https://www.gitlab.com/psmisc/psmisc])
 AM_INIT_AUTOMAKE([foreign 1.11 subdir-objects -Wall -Wno-portability tar-pax no-dist-gzip dist-xz])
+AM_MAINTAINER_MODE
 AM_SILENT_RULES([yes])
 AC_CONFIG_SRCDIR([src/comm.h])
 AC_CONFIG_HEADERS([config.h])

# source
http://downloads.sourceforge.net/psmisc/psmisc-23.6.tar.xz
